CMakeLists.txt

```diff
@@ -334,6 +334,11 @@ if(ALLEGRO_RASPBERRYPI)
    set(ALLEGRO_EXCLUDE_GLX 1)
 endif(ALLEGRO_RASPBERRYPI)
 
+if(EMSCRIPTEN)
+    set(GL_AUTO_BUILD_TYPE "gles2+")
+    set(ALLEGRO_SDL on)
```

Comment:
I'm a little uncomfortable about turning this on silently. Could we force the user to turn on SDL manually and check here that it is set?

Reply:
Makes sense :+1: